sycl/test/aie/ranged_buffer.cpp

```diff
@@ -0,0 +1,29 @@
+// REQUIRES: aie
+
+// RUN: %aie_clang %s -o %t.bin
+// RUN: %if_run_on_device %run_on_device %t.bin > %t.check 2>&1
+// RUN: %if_run_on_device FileCheck %s --input-file=%t.check
+
+#include "aie.hpp"
+
+int main() {
+  constexpr std::size_t size = 10;
+  aie::device<1, 1> dev;
+  aie::buffer<int> buff;
+  for (int i = 0; i < size; i++)
+    buff.push_back(i);
+  aie::queue q(dev);
+  q.submit_uniform([&](auto& ht) {
+    aie::accessor acc = aie::buffer_range(ht, buff)
+                            .read_range(0, buff.size())
+                            .write_range(0, buff.size());
```

Comment:
So the API is asymmetric read and write. That is interesting.
But you need to update the example.
Perhaps the `iota` initializing the lower half of the buffer and the kernel compute the higher half from the lower one.

Reply:
No taker?